debug.go

```diff
@@ -60,10 +69,200 @@ func newDebugger(args []string) *debugger {
 // high enough to print the message.
 func (d *debugger) debugf(level int, format string, v ...interface{}) {
 	if d.Level >= level {
-		d.Log.Printf(format, v...)
+		if d.Format == "mnl" {
+			for _, iface := range v {
+				if msg, ok := iface.(Message); ok {
+					nlmsgFprintf(d.Log.Writer(), msg)
+				}
```

Comment:
Shouldn’t we call `d.Log.Printf()` in the else branch here? If not, can you add a comment explaining why this code path only handles values of type `Message`?

Reply:
oh, it was an oversight,

Reply:
I refactored the code to remove one indentation layer and added a switch statement if there is need to add different output formats, I can see pyroute2 format is also used in different places